.github/workflows/pull.yml

```diff
@@ -1,7 +1,7 @@
 name: pull
 
 on:
-  pull_request:
+  # pull_request:
```

Comment:
this disables everything except lint. you can still trigger workflows using a label

Reply:
once done, you can remove this, reopen the PR, all ci jobs should be scheduled